starter-code/css/main.css

```diff
@@ -4,7 +4,124 @@ dark blue: #06365f
 light blue: #1c5380
 */
 
+/* ---- Reset ---- */
+
+html {
+  box-sizing: border-box ;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+
+
+/* ---- Typografie ---- */
+
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
+  margin: auto;
```

Comment:
better set boddy paddings and margins to 0, and then you create a container with everything inside.